contrib/endpoints/src/api_manager/config_manager.cc

```diff
@@ -47,6 +47,11 @@ ConfigManager::ConfigManager(
     }
   }
 
+  if (global_context_->server_config()->has_init_service_configs()) {
```

Comment:
not copy it in constructor.   only set the current_rollout_id_ in line 81 of api_manager_impl.cpp after DeployConfigs() call.